packages/webpack-cli/lib/bootstrap.js

```diff
@@ -49,14 +49,21 @@ async function runCLI(cli, commandIsUsed) {
             // if the unknown arg starts with a '-', it will be considered
             // an unknown flag rather than an entry
             let entry;
-            if (parsedArgs.unknownArgs.length === 1 && !parsedArgs.unknownArgs[0].startsWith('-')) {
-                entry = parsedArgs.unknownArgs[0];
-            } else if (parsedArgs.unknownArgs.length > 0) {
-                parsedArgs.unknownArgs
-                    .filter((e) => e)
-                    .forEach((unknown) => {
-                        logger.warn('Unknown argument:', unknown);
+            if (parsedArgs.unknownArgs.length > 0 && !parsedArgs.unknownArgs[0].startsWith('-')) {
```

Comment:
This is not a way to go, set multipleValues in cliFlags instead

Reply:
That way we will be able to support  `webpack --entry a.js --entry b.js`

The idea is to support multiple entries by default without passing `--entry` i.e. `webpack a.js b.js c.js`

Reply:
Or we can support , need discussion for same.